cg/meta/orders/api.py

```diff
@@ -69,7 +69,9 @@ def _submit_rml(self, order: dict) -> dict:
 
     def _submit_pools(self, order):
         status_data = self.pools_to_status(order)
-        project_data, lims_map = self.process_lims(order, order["samples"])
+        project_data, lims_map = process_lims(
+            lims_api=self.lims, data=order, samples=order["samples"]
```

Comment:
Why send both the full order and part of the order (order["samples"])?
I suggest we find ourselves a more descriptive argument name than "data"

Reply:
Good suggestions, changed accordingly